specs/core/0_beacon-chain.md

```diff
@@ -1749,15 +1747,14 @@ Also perform the following updates:
 
 * Set `state.validator_registry_latest_change_slot = state.slot`.
 * Set `state.shard_committees_at_slots[:EPOCH_LENGTH] = state.shard_committees_at_slots[EPOCH_LENGTH:]`.
-* Set `state.shard_committees_at_slots[EPOCH_LENGTH:] = get_new_shuffling(state.next_seed, state.validator_registry, next_start_shard)` where `next_start_shard = (state.shard_committees_at_slots[-1][-1].shard + 1) % SHARD_COUNT`.
-* Set `state.next_seed = state.randao_mix`.
+* Set `state.shard_committees_at_slots[EPOCH_LENGTH:] = get_new_shuffling(state.latest_randao_mixes[(block.slot - CYCLE_LENGTH) % LATEST_BLOCK_ROOTS_LENGTH], state.validator_registry, next_start_shard)` where `next_start_shard = (state.shard_committees_at_slots[-1][-1].shard + 1) % SHARD_COUNT`.
```

Comment:
If we're going to do that, why tie `SEED_LOOK_AHEAD` to being a multiple of `CYCLE_LENGTH` (should be `EPOCH_LENGTH`; I fixed); it could also be half an epoch. So if the variables should be separated, just use `state.slot - SEED_LOOK_AHEAD`.

Reply:
You're right.
Do you see this as a param likely to be tuned or do you want to keep it CYCLE_LENGTH for now??